examples/build.gradle

```diff
@@ -1,12 +1,14 @@
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
-    api(project(":api"))
+    api project(":api")
+    implementation project(":visualization")
 }
 
 dependencies {
     compile 'org.apache.logging.log4j:log4j-api:2.14.0'
     compile 'org.apache.logging.log4j:log4j-core:2.14.0'
     compile 'org.apache.logging.log4j:log4j-slf4j-impl:2.14.0'
```

Comment:
Did you see some conflicts with logging versions? do we need exactly these versions? could you please explain this?

Reply:
I don't understand your questions. What kind of conflicts do you mean?

Reply:
I just changed the 
```
api(project(":api"))
```
to 
```
api project(":api")
```
to make concise gradle syntax in these imports (look how you use the "implementation smth" not "implementation(smth)"  )

Reply:
I had some strange warnings, running your branch locally, will share it later